codes/classical/q-ary_digits/gabidulin.yml

```diff
@@ -0,0 +1,29 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: gabidulin
+physical: q-ary_digits
+logical: q-ary_digits
+
+name: 'Gabidulin or rank code'
+introduced: '\cite{manual_citation_minilatex:{E. M. Gabidulin, "Theory of Codes with Maximum Rank Distance", Problemy Peredachi Informacii, Volume 21, Issue 1, \emph{3–16} (1985)}}'
```

Comment:
`\cite{manual:...}` does not work

Reply:
Try \cite{manual:{E. M. Gabidulin, "Theory of Codes with Maximum Rank Distance", Problemy Peredachi Informacii, Volume 21, Issue 1, 3–16 (1985)}}

Reply:
Also \cite{doi:10.1109/18.75248}, who discovered them independently

Reply:
updated